superset/commands/sql_lab/execute.py

```diff
@@ -141,6 +144,7 @@ def _run_sql_json_exec_from_scratch(self) -> SqlJsonExecutionStatus:
         self._execution_context.set_database(self._get_the_query_db())
         query = self._execution_context.create_query()
         self._save_new_query(query)
+        db.session.flush()
```

Comment:
@michael-s-molina here's a better example where `db.session.flush()` is called in the command given that it's not invoked by the underlying DAO.